cloudos/jobs/job.py

```diff
@@ -231,6 +238,7 @@ def convert_nextflow_to_json(self,
                 "optim": "test"
             },
             "revision": revision_block,
+            "profile": nextflow_profile,
```

Comment:
What will be default value setting here in case no `--nextflow-profile` given

Reply:
Internally, python assign it `None` and it can be correctly handled by the API as not giving any profile generates the correct job call
![Screenshot from 2022-06-16 14-12-29](https://user-images.githubusercontent.com/45285897/174082483-ff38c2f8-4ac3-4eb7-b738-c1facf89f8a6.png)
: